app/src/main/java/one/credify/sdk/sample/BaseActivity.kt

```diff
@@ -1,19 +1,57 @@
 package one.credify.sdk.sample
 
 import android.app.Dialog
+import android.content.Intent
 import android.os.Bundle
+import android.view.MenuItem
+import android.widget.Toast
 import androidx.appcompat.app.AlertDialog
 import androidx.appcompat.app.AppCompatActivity
+import androidx.appcompat.widget.Toolbar
 
 abstract class BaseActivity : AppCompatActivity() {
     private lateinit var dialog: Dialog
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
 
+        onGetInputData(intent)
+
         createDialog()
     }
 
+    override fun onOptionsItemSelected(item: MenuItem): Boolean {
```

Comment:
`onOptionItemSelected`

Reply:
@ngo275 This method provided by Android. I cannot change it.

Reply:
Oh OK